pages/windows/powershell.md

```diff
@@ -0,0 +1,32 @@
+# powershell
+
+> Command-line shell and scripting language designed especially for system administration.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/powershell>.
+
+- Start a Windows PowerShell session in a Command Prompt window:
+
+`PowerShell`
```

Comment:
Are you sure it's got capital letters in the name? Generally most commands are in lowercase. If you have a way to find out, we should use the casing of the original filename of the command on disk - since while on Windows casing doesn't matter, but on other OSes it does (and it's good to be consistent across all OSes). If not, then if possible it's probably best to go with all lowercase here.

Reply:
Just had a look on Windows 10, and it is lowercase (`powershell.exe`). So should be `powershell`. 👍🏻

Reply:
@sbrl Oh, the doc page says `PowerShell.exe` maybe is outdated. And now is lowercase to be compatible with `powershell` in linux. Should we check the compatibility to move to the `common` folder?

Reply:
I don't think `powershell` is compatible with Linux. PowerShell Core is, but that uses the name `pwsh`. 🤔

Reply:
@owenvoke Oh, cool. I didn't use it. Should we use the lowercase `powershell` then?

Reply:
Yes. 👍🏻 And leave it under the `windows` directory. 🙂

Reply:
Hi! I'm kinda confused. Should I change every related command to lowercase too?

Reply:
@byt3h3ad, those are fine, just the binary (`powershell`) should be lowercased. Windows is weird like that. 😬

Reply:
Windows is case insensitive, e.g. `POWERSHELL` will work too. I think the real case is when inside powershell you type `powersh`+<tab>. Or look for the `.exe` file in the Windows Explorer

Reply:
I see. Thanks for clearing it for me 😁

Reply:
Thanks, @byt3h3ad! Looks like everyone else beat me to it explaining this one :P

@owenvoke: Oh, that's interesting. It has a completely different name on linux? Wow, that's not confusing at all :sweat_smile: